test/test.go

```diff
@@ -505,3 +505,15 @@ func (h *Helper) GetCommit(repo string) string {
 	}
 	return strings.TrimSpace(string(out))
 }
+
+func (h *Helper) GetLatestTag(repo string) string {
+	repoPath := h.Path("pkg/dep/sources/https---" + strings.Replace(repo, "/", "-", -1))
```

Comment:
This approach binds us to internal storage implementation details of the `gps.SourceManager`. It's preferred to either use the abstractions provided directly (`SourceManager.ListVersions`), or, as I suggested in #228 (sorry, perhaps after you got started on this PR 😞 ) just rely on a static upstream source that's safe to hardcode.